web_widget_domain_editor_dialog/static/src/js/widget_domain_editor_dialog.js

```diff
@@ -0,0 +1,74 @@
+/* Copyright 2019 Tecnativa - David Vidal
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+
+odoo.define("web_widget_domain_editor_dialog.DomainEditorDialog", function (require) {
+    "use strict";
+
+    const core = require("web.core");
+    const view_dialogs = require("web.view_dialogs");
+    const Domain = require("web.Domain");
+    const _t = core._t;
+
+    const DomainEditorDialog = view_dialogs.SelectCreateDialog.extend({
+        init: function () {
+            this._super.apply(this, arguments);
+            const _this = this;
+            this.options = _.defaults(this.options, {
+                dynamicFilters: [
+                    {
+                        description: _.str.sprintf(_t("Selected domain")),
+                        domain: Domain.prototype.stringToArray(
+                            _this.options.default_domain
+                        ),
+                    },
+                ],
+            });
+        },
```

Comment:
```suggestion
        init: function () {
            this._super.apply(this, arguments);
            this.options = _.defaults(this.options, {
                dynamicFilters: [
                    {
                        description: _t("Selected domain"),
                        domain: Domain.stringToArray(this.options.default_domain),
                    },
                ],
            });
        },
```
Pretty sure that should work just as well